libs/methodologies/bold/testing/src/stubs/document.stubs.ts

```diff
@@ -132,3 +132,31 @@ export const stubOfferCertificatesDocument = (
     subtype: DocumentSubtype.GROUP,
     type: DocumentType.CREDIT_CERTIFICATES,
   });
+
+export const stubMethodologyDefinitionDocument = (
+  partialMethodologyDefinitionDocument?: PartialDeep<Document>,
```

Comment:
pra simplificar a leitura e facilitar possíveis alterações no nome dos métodos, uma opção seria omitir o nome da função do nome do argumento. Que aí se a função for renomeada, por exemplo, não precisaria renomear o arg tbm (mesma sugestão para as funções seguintes)
```suggestion
  partialDocument?: PartialDeep<Document>,
```